addons/l10n_pl/data/template/account.account-pl.csv

```diff
@@ -1,92 +1,77 @@
 "id","name","code","account_type","tag_ids","reconcile","name@pl"
-"chart01010000","Proprietary land and rights of perpetual usufruct of land","01.010.000","asset_fixed","","False","Grunty własne i prawa wieczystego użytkowania gruntów"
```

Comment:
![image](https://github.com/odoo/odoo/assets/109144554/3123f313-c6fc-43d0-b0dd-bd2b07581960)
There are some duplicate names, maybe we can try to avoid them ?